src/components/asset-list/RecyclerAssetList2/core/Contexts.ts

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react';
+import { Animated as RNAnimated } from 'react-native';
+
+import { useDeepCompareMemo } from 'use-deep-compare';
+
+export const RecyclerAssetListContext = React.createContext<
+  Record<string, object>
```

Comment:
Should this be `CellTypes` rather than `object`?

Reply:
correct, thanks